### DISCUSSION TITLE
突然感觉绳网变米网了

### DISCUSSION THREAD

**Discussion by fashibuding1214** [share121/inter-knot#4568] [其他]
![Uploading mmexport1722675916053.jpg…]()
今天刚下绳网app，一看发现还有原铁崩分区，虽然我也玩原铁，但有种绳网变米网的感觉(Ｔ▽Ｔ)
其实还是比较想要一个独属于zzz的论坛来着

**Comment by fashibuding1214** [comment]
不过知道这个绳网的应该基本都是从zzz过来，应该不会有人特地来绳网，是为了发原铁崩的东西吧。虽然大家玩的游戏多少会重合其他米游，但可以直接在类似综合版的板块里发出来，特地弄原特崩的分区出来会不会有点没必要而且显得不伦不类(　ﾟ 3ﾟ)（思考）？
可以像x岛（以前的a岛）一样直接弄一个大综合版，这样大伙基本都会往综合里发贴。然后再添加一些比如委托，攻略，二创之类的次级版块。个人感觉这样既不会破坏绳网本身的特点也允许发各式各样的内容，保持了绳网作为zzz衍生产物同时也更符合绳网的气质，比较进绳网的都是绳匠嘛( ﾟ∀。)。如果实在想看特定内容，可以添加一个标签功能，可以在搜索功能里检索的那种。
个人感觉可能这样会更好，只属于个人建议，希望绳网越发展越好( ﾟ∀。)大伙怎么想

**Comment by swyh750810** [comment]
我记得这不是有分区吗？

**Reply by fashibuding1214** [reply]
就是说在zzz的衍生论坛里特地划出原铁崩分区有点怪，就总觉得不是绳网而是米网了(　ﾟ 3ﾟ)

**Reply by The-Place-Where-Winter-Sieeps** [reply]
呃呃呃，有什么区别吗
基本上都是B站来的吧
而米游玩家基本上高度重合
玩米游，推送的视频都差不多
所以就不止是zzz玩家？
（如果不是就当我没说
<img src="https://img-static.mihoyo.com/communityweb/upload/8b198c0a467614f263e08593bd4b000d.png" alt="安比_无语" width="75" >

**Reply by fashibuding1214** [reply]
嘛……可能我比较喜欢有代入感一些。
反正大伙也多少也都玩别的米游就算没有分区也会发，但特地弄出原铁崩的分区，就好像我明明是冲着绳网来的，进来一看，还特地划分了四个米游的讨论分区，本来是想体验绳网的镜像论坛的，现在感觉是变成米游的论坛了🤨就感觉，很出戏，小怪。
我现在是其他米游淡了，主玩绝区零，就有这样的感觉。
更新后发现了这个分区，就作者大佬弄出这个镜像论坛的目的在哪。如果只是搞着玩玩，那都挺好。如果是为了给予米游玩家们一个新的纯净的讨论环境那这样也挺好。如果是想专门为了给个绝区零玩家弄个这样的论坛那感觉改一改会好点。不过看作者大佬头像应该更偏向第二个😂

**Comment by nzj2004** [comment]
游戏玩的多很正常，至少这种有一定门槛的网站，会比米游社舒服一点，这个界面更舒服一些

**Comment by aino2333** [comment]
哎呀这种事很正常的。米友社也有大别野可以发其他游戏啦<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/c0ee663156c5bb4dc518341f2fd231dd_2752194756869066332.png" alt="驭空 叹气" width="75" >
